wrapper/src/main/java/software/amazon/jdbc/plugin/failover/FailoverConnectionPlugin.java

```diff
@@ -339,7 +341,8 @@ private void initSettings() {
         FAILOVER_WRITER_RECONNECT_INTERVAL_MS.getInteger(this.properties);
     this.failoverReaderConnectTimeoutMsSetting =
         FAILOVER_READER_CONNECT_TIMEOUT_MS.getInteger(this.properties);
-    this.enableFailoverStrictReaderSetting = ENABLE_FAILOVER_STRICT_READER.getBoolean(this.properties);
+    this.failoverModeSetting = FailoverMode.fromValue(FAILOVER_MODE.getString(this.properties));
```

Comment:
Would it work to remove the failoverModeSetting variable and just set set `this.failoverMode` to the value in the props? It looks like we set failoverMode to failoverModeSetting in initHostProvider anyways so I'm not sure if we need both variables